orderer/consensus/etcdraft/membership.go

```diff
@@ -105,6 +114,102 @@ func ComputeMembershipChanges(oldMetadata *etcdraft.BlockMetadata, oldConsenters
 	return result, nil
 }
 
+func validateConsenterTLSCerts(c *etcdraft.Consenter, ordererConfig channelconfig.Orderer) error {
+	clientCert, err := parseCertificateFromBytes(c.ClientTlsCert)
+	if err != nil {
+		return errors.Wrap(err, "parsing tls client cert")
+	}
+	serverCert, err := parseCertificateFromBytes(c.ServerTlsCert)
+	if err != nil {
+		return errors.Wrap(err, "parsing tls server cert")
+	}
+
+	for _, org := range ordererConfig.Organizations() {
```

Comment:
why do you need to loop around the organizations? Just construct the TLS CA cert pools from the union of all the CA certs over all MSPs, and give it a go. The x509 library should be able to find a validation path if such exists.

Reply:
I did this due to the assumption that we'd want the client and server TLS certs to be valid under the same organization's MSP. I can change this if that assumption is wrong.

Reply:
In the end of the day, the check should validate the current behavior of TLS validation - which actually aggregates all TLS root/intermediate CA certificates and also adds the ones in the local config.

Reply:
Combined into a single set of VerifyOptions with all TLS root/intermediate CA certs.